skimage/data/__init__.py

```diff
@@ -328,6 +329,31 @@ def rocket():
     return load("rocket.jpg")
 
 
+def satellite():
+    """Launch photo taken by satellite.
+
+    This is a satellite imagery showing an urban area.
+
+    Notes
+    -----
+    This image was downloaded from USGS public data portal:
+    https://earthexplorer.usgs.gov/
+    The data is in US public domain. For copyright info, go to:
+    https://www2.usgs.gov/laws/info_policies.html
+
+    Courtesy of U.S. Geological Survey
+    Department of the Interior/USGS
+
+    Product name: High Resolution Orthoimagery
+    Acquisition Date: 27-Oct-14
+
+    Returns
+    -------
```

Comment:
Could you please add something after Returns, as in the other functions of the module?